crates/bevy_input/src/touch.rs

```diff
@@ -213,14 +285,20 @@ impl Touches {
         };
     }
 
+    /// Clears the `just_pressed`, `just_released`, and `just_cancelled` collections.
```

Comment:
If I remember correctly there was a PR that tried to clear the pressed collection and it broke everything. I would had a short explanation as to why pressed is not being cleared here.